index.js

```diff
@@ -115,3 +64,108 @@ app.on('activate', () => {
         createWindow();
     }
 });
+
+function getArgs() {
+    // Get the parameters
+    let failed = false;
+    var argv = yargs(hideBin(process.argv))
+        .help("h").alias("h", "help")
+        .usage("Usage: $0 [<options>]")
+        .example("$0 -b zepto.bin", "Start the emulator and execute zepto.bin in the emulator")
+        .option('rom', {
+            type: 'string',
+            alias: 'r',
+            description: 'Select a prebuild romdisk or a local image',
+            nargs: 1,
+        })
+        // TODO: 
+        // .option('binary', {
+        //     type: 'string',
+        //     alias: 'b',
+        //     description: 'Load a executable binary into ZOS',
+        //     nargs: 1,
+        // })
+        // .implies("binary", "rom")
+        .option('breakpoint', {
+            type: 'string',
+            alias: 'B',
+            description: 'Set a breakpoint',
+            array: true,
+            nargs: 1,
+        })
+        .array("breakpoint")
+        .option('map', {
+            type: 'string',
+            alias: 'm',
+            description: 'Load a map file',
+            nargs: 1,
+        })
+        .option('eeprom', {
+            type: 'string',
+            alias: 'e',
+            description: 'Load a file into EEPROM',
+            nargs: 1,
+        })
+        .option('cf', {
+            type: 'string',
+            alias: 'c',
+            description: 'Load a file into CompactFlash',
+            nargs: 1,
+        })
+        .strictOptions()
+        .exitProcess(false)
+        .fail((msg, err, yargs) => {
+            console.log(msg);
+            // if(error) throw err;
+            console.log(yargs.help());
+            failed = true;
+        })
+        .parse();
+    if(failed) return false;
+    return argv;
+}
+
+function parseArgs(argv) {
+    if (argv.rom) {
+        let img_regex = /^v[0-9]*\.[0-9]*\.[0-9]*/;
+        if (img_regex.test(argv.rom)) {
+            if (fs.existsSync(`./roms/${argv.rom}.img`)) {
+                argv.rom = `./roms/${argv.rom}.img`;
+            }
+            else {
+                // TODO: get image by auto and add `--rom latest`
+                console.log("please run `pnpm get-prebuilt` to get latest images");
+            }
+        }
+        setTimeout(()=>{
```

Comment:
Is this timeout added to give some time to the page to load?
Is there any onload or similar callback instead?